integreat_cms/cms/templates/contacts/contact_card.html

```diff
@@ -0,0 +1,62 @@
+{% load svg_tags %}
+{% load static %}
+{% spaceless %}
+    <div contenteditable="false"
+         data-contact-id="{{ contact.pk }}"
+         data-contact-url="{{ contact.full_url }}"
+         class="contact-card inline-block box-border p-4 rounded bg-no-repeat cursor-default text-[initial] shadow-md shadow-black decoration-[initial] min-w-[50%]">
+        <a href={{ contact.full_url }} class="marker-link">Contact</a>
+        {% if contact %}
+            {% if contact.name or contact.point_of_contact_for %}
+                <h4>
+                    {% if contact.name %}
+                        <span class="notranslate" dir="ltr" translate="no">{{ contact.name }}</span>
```

Comment:
Sorry that I missed this in the first review. 
Is the idea of adding notranslate to each text individually to make it possible that `contact.point_of_contact_for` gets translated? Because right now we overwrite the translated contact card anyways, so the possible translations get lost. So maybe we could just have a single notranslate at the root div?